src/Analyzer/Resolve/IdentifierResolver.cpp

```diff
@@ -420,6 +420,11 @@ QueryTreeNodePtr IdentifierResolver::tryResolveTableIdentifierFromDatabaseCatalo
     if (!storage)
         return {};
 
+    if (storage->hasExternalDynamicMetadata())
+    {
+        storage->updateExternalDynamicMetadata(context);
+    }
```

Comment:
```suggestion
        storage->updateExternalDynamicMetadata(context);
```

We usually don't write braces with single expression in `if/while/for/...`. Let's be consistent (and the same for similar places in this PR). It's optional, but imho it's easier to read when everything is in the same style

Reply:
I think that such braces make the code safer (in case you need to add the second line to the block, you won't forget to put it in the block). Also I think that style checks and corrections should be done automatically, so I try to fix only critical style issues